rpc.go

```diff
@@ -1,7 +1,28 @@
 package rpc
 
-func Ping(params interface{}, id ...interface{}) string {
+import (
+	"encoding/json"
+)
+
+// RPC struct
+type RPC struct {
+	base
```

Comment:
Seongchan told me that when you used "embedding", the embedded type's function is exposed. The RPC type should not expose "id" and "call" functions. Let's do not use the "embedding" here.

Reply:
https://github.com/CodeChain-io/codechain-rpc-go/pull/4/commits/db87439a49f1c1a5735e5d8c2541468c75fcda18